server/app.js

```diff
@@ -1,32 +1,37 @@
+/* eslint-disable import/no-dynamic-require */
 const createError = require('http-errors');
 const express = require('express');
 const path = require('path');
-const cookieParser = require('cookie-parser');
-const logger = require('morgan');
-
-const indexRouter = require('./routes/index');
-const usersRouter = require('./routes/users');
+const fs = require('fs');
+require('dotenv').config();
+const mongoose = require('mongoose');
+const pino = require('express-pino-logger')();
+const logger = require('./utils/logger');
 
 const app = express();
+const modelDirPath = path.join(__dirname, 'models');
+const routeDirPath = path.join(__dirname, 'routes');
 
-// view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'jade');
-
-app.use(logger('dev'));
+app.use(pino);
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
+fs.readdirSync(modelDirPath).forEach((file) => {
```

Comment:
Please add empty lines for splitting code into logical blocks

Reply:
Done as suggested!